backend/routes/api/jobs.js

```diff
@@ -67,7 +69,8 @@ router.put("/:id", function (req, res, next) {
     requirements: req.body.requirements,
     details: req.body.details,
     contact: req.body.contact,
-    matches: req.body.matches,
```

Comment:
why is this deleted

Reply:
Ya why is this deleted

Reply:
👀I just changed the order, trust me its there